render-release-notes.sh

```diff
@@ -0,0 +1,174 @@
+#!/bin/bash
+
+me=$(basename $0)
+
+function showUsage() {
+	cat<<-EOF
+Usage: $me --installer <msi|inno|nuget>
```

Comment:
I am not a fan of making this script aware of certain installers. Rather, the scripts in the respective subdirectories should know where/how they want things done.